setup.cfg

```diff
@@ -65,14 +65,14 @@ filterwarnings =
     error
 
 [coverage:run]
-branch = true
+parallel = true
```

Comment:
You'll want to keep `branch` and not `parallel`. Branch coverage is important, and parallel mode should be enabled at the call in the tox config (and isn't needed in the config you've written since there's only one env).

Reply:
For some reasons I do not understand, without `parallel = true` the coverage report paths are `.tox/py38/.../src/wtforms` instead of `src/wtforms/`

https://hynek.me/articles/testing-packaging/

Reply:
Ah, this is one of the reasons I got frustrated with coverage. :smile: